.github/workflows/test.yaml

```diff
@@ -0,0 +1,35 @@
+name: Test
+
+on: [pull_request]
+
+jobs:
+  Bygg:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+      - name: Setup Java environment
+        uses: actions/setup-java@v4
+        with:
+          java-version: 21
+          distribution: temurin
+      - name: Cache Gradle wrapper
+        uses: actions/cache@v4
+        with:
+          path: ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-wrapper-${{ hashFiles('gradle/wrapper/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-wrapper-
+      - name: Cache Gradle packages
+        uses: actions/cache@v4
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-cache-${{ hashFiles('build.gradle') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-cache-
+      - name: Test
+        uses: gradle/gradle-build-action@v3
+        with:
+          arguments: test --console=plain
+        env:
+          ORG_GRADLE_PROJECT_githubPassword: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
`gradle/gradle-build-action@v3` bør erstattes med `gradle/actions/setup-gradle@v4`. Da må du ha et egen steg for å kjøre test-kommandoen, og kan/bør fjerne begge stegene med `action/cache`. Se https://github.com/navikt/helsearbeidsgiver-inntektsmelding/blob/main/.github/workflows/test.yml.